src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -45,7 +45,7 @@ public AddCommand parse(String args) throws ParseException {
         Address address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get());
         Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
 
-        Person person = new Person(name, phone, email, address, tagList);
+        Person person = new Person(name, phone, email, address, tagList, -1);
```

Comment:
You could consider overloading the constructor method of Person here, such that the ID is initialized to -1 if not specified. This way, your code could be clearer, and the original constructor method signature can still be used.

Reply:
Thanks for the feedback! I will change it right now.